consensus/src/consensus.rs

```diff
@@ -119,6 +173,12 @@ pub struct Consensus<PublicKey: VerifyingKey, ConsensusProtocol> {
 
     /// Metrics handler
     metrics: Arc<ConsensusMetrics>,
+
+    /// The latest committed round of each validator.
+    last_committed: HashMap<PublicKey, Round>,
+
+    /// Certificates to load into the DAG
+    certs: HashMap<CertificateDigest, Certificate<PublicKey>>,
```

Comment:
I'm confused: you seem to read those from the certificate store when constructing `ConsensusState` in the first place, why do you need those here as well?

Reply:
removing this